ccpp/data/GFS_typedefs.F90

```diff
@@ -6697,13 +6728,15 @@ subroutine radtend_create (Radtend, IM, Model)
     allocate (Radtend%coszen (IM))
     allocate (Radtend%tsflw  (IM))
     allocate (Radtend%semis  (IM))
+    allocate (Radtend%ext550 (IM,Model%levs))
```

Comment:
Does this only need to be allocated when `model%rrfs_sd = T`? If so, and you decide to conditionally-allocate, be sure to  change the `active` attribute in the metadata too.

Reply:
This is used in several modules regardless of whether smoke is enabled.